src/psyclone/psyad/tl2ad.py

```diff
@@ -75,6 +75,7 @@ def generate_adjoint_str(tl_fortran_str, active_variables,
     :param Optional[str] api: the PSyclone API in use, if any.
     :param Optional[bool] create_test: whether or not to create test code for
         the adjoint kernel.
+    :param Optional[str] test_filename: name of the adjoint test filename.
```

Comment:
"name to use for the file containing the created adjoint test"

Reply:
Addressed in 73d818f2ad29ad1c7bbf606048890bb5ba123ce2 and 83906e0c8063317fed7f77f5c48549104508538f.